### DISCUSSION TITLE
Configuração Webhook

### DISCUSSION THREAD

**Discussion by gilsonjoanelo** [bacen/pix-api#548] [General] [answered]
Estou ativando a integração PIX no sistema da empresa na qual trabalho, porém, estou me debatendo com relação a forma que é retornada a notificação do pagamento do PIX gerado, como registrar, remover ou consultar o webhook isso é tranquilo. A dúvida é, qual a forma de requisição o Banco irá fazer para o site, é via POST (provavelmente), GET, quais parâmetros esse endpoint precisa ter? tem isso em algum local? 

encontrei a documentação oficial do Bacen, não encontrei nada com relação a forma que deve ser implementado esse Endpoint. Fiz cadastro como Dev nos bancos, Santander, Banco do brasil e Bradesco, em nenhum deles encontrei a informação.

**Comment by rafaelchagasb** [comment, marked as answer]
@gilsonjoanelo Dá uma olhada no PUT webhook. Lá tem uma aba chamada callback. 

Lá tem os detalhes de como definir a URL do webhook e o formato do objeto.

![image](https://user-images.githubusercontent.com/724699/227361758-aae2a819-441b-4366-b38e-146db490fdbb.png)

**Reply by gilsonjoanelo** [reply]
Obrigado pelo retorno,
